lib/components/narrative/metro/default-route-renderer.tsx

```diff
@@ -4,22 +4,25 @@ import styled from 'styled-components'
 
 // TODO: This should not be a section, but using div/span
 // doesn't allow us to style it from within the RouteBlock
-const Block = styled.section<{ color: string }>`
+const Block = styled.section<{ color: string; hideTopBorder?: boolean }>`
   background: #${(props) => props.color}1A;
-  padding: 3px 7px;
-  border-top: 5px solid #${(props) => props.color};
   border-radius: 5px;
-  text-align: center;
+  border-top: 5px solid #${(props) => props.color};
   display: inline-block;
   max-width: 12vw;
+  padding: 3px 7px;
+  text-align: center;
+  white-space: break-spaces;
+
+  ${(props) => props.hideTopBorder && 'border-top: none'}
 `
 
 type RouteRendererProps = {
-  leg: Leg
+  leg: Leg & { hideTopBorder?: boolean }
 }
 
 const DefaultRouteRenderer = ({ leg }: RouteRendererProps): JSX.Element => (
-  <Block color={leg.routeColor || '333333'}>
+  <Block color={leg.routeColor || '333333'} hideTopBorder={leg.hideTopBorder}>
```

Comment:
It's definitely a hack... The issue comes from the fact that the `RouteRenderer` component doesn't really exist if that makes sense. Since it's pulled in from context, we can't change the props. I could try to find the exact detailed reason why this was done but it was a while ago and I remember I spent a good amount of time trying to figure out how to avoid it.